src/components/Dropdown/Dropdown.tsx

```diff
@@ -55,6 +57,7 @@ export function Dropdown(props: PropsWithChildren<DropdownProps>): JSX.Element {
   const containerRef = useRef<HTMLDivElement>(null);
   const screenReaderUUID: string = useMemo(() => uuid(), []);
   const [screenReaderKey, setScreenReaderKey] = useState<number>(0);
+  const [readAutocomplete, setReadAutocomplete] = useState<boolean>(false);
   const [childrenWithDropdownItemsTransformed, items] = getTransformedChildrenAndItemData(children);
```

Comment:
Could we use `items.length` instead of a new `numItems` prop? I think in the future we want to get rid of `getTransformedChildrenAndItemData` but until we do that I think we don't need an extra prop

Reply:
ooh ok